assets/src/edit-story/components/panels/textStyle/padding.js

```diff
@@ -103,14 +164,22 @@ function PaddingControls({ selectedElements, pushUpdateForObject }) {
         data-testid="padding.vertical"
         suffix={_x('V', 'The Vertical padding', 'web-stories')}
         value={padding.vertical}
-        onChange={(value) => handleChange({ vertical: value })}
+        onChange={(value) =>
+          pushUpdateForObject(
+            'padding',
+            {
+              vertical: value,
+            },
+            DEFAULT_PADDING
+          )
+        }
       />
-    </Row>
+    </>
   );
 }
 
-PaddingControls.propTypes = {
-  selectedElements: PropTypes.array.isRequired,
+UnlockedPaddingControls.propTypes = {
+  padding: PropTypes.shape().isRequired,
```

Comment:
In case of shape, I think we need particular format of this shape object.
`.shape({ vertical: PropTypes.number, horizontal: PropTypes.number })` for example?

Reply:
Done.